tsl/test/isolation/expected/compression_chunk_race.out

```diff
@@ -0,0 +1,131 @@
+Parsed test spec with 3 sessions
+
+starting permutation: s3_lock_compression s3_lock_decompression s1_compress s2_compress s3_unlock_compression s1_decompress s2_decompress s3_unlock_decompression
+step s3_lock_compression: 
+    SELECT debug_waitpoint_enable('compress_chunk_impl_start');
+
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step s3_lock_decompression: 
+    SELECT debug_waitpoint_enable('decompress_chunk_impl_start');
+
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step s1_compress: 
+   SELECT compression_status FROM chunk_compression_stats('sensor_data');
+   SELECT count(*) FROM (SELECT compress_chunk(i, if_not_compressed => true) FROM show_chunks('sensor_data') i) i;
+   SELECT compression_status FROM chunk_compression_stats('sensor_data');
+   SELECT count(*) FROM sensor_data;
+ <waiting ...>
+step s2_compress: 
+   SELECT compression_status FROM chunk_compression_stats('sensor_data');
+   SELECT count(*) FROM (SELECT compress_chunk(i, if_not_compressed => true) FROM show_chunks('sensor_data') i) i;
+   SELECT compression_status FROM chunk_compression_stats('sensor_data');
+   SELECT count(*) FROM sensor_data;
+   RESET client_min_messages;
+ <waiting ...>
+step s3_unlock_compression: 
+    -- Ensure that we are waiting on our debug waitpoint and one chunk
+    -- Note: The OIDs of the advisory locks are based on the hash value of the lock name (see debug_point_init())
+    --       compress_chunk_impl_start = 3379597659.
+    -- 'SELECT relation::regclass, ....' can not be used, because it returns a field with a variable length
+    SELECT locktype, mode, granted, objid FROM pg_locks WHERE not granted AND (locktype = 'advisory' or relation::regclass::text LIKE '%chunk') ORDER BY relation, locktype, mode, granted;
+    SELECT debug_waitpoint_release('compress_chunk_impl_start');
+
+locktype|mode     |granted|     objid
+--------+---------+-------+----------
+relation|ShareLock|f      |          
+advisory|ShareLock|f      |3379597659
```

Comment:
This will result in flaky tests

Reply:
The results are filtered and sorted. The `objid` is also stable since it depends on the name of the debug waitpoint hash value. So, this should only change if we change the hash function. What have I missed? Are you concerned about additional locks that could show up?

Reply:
I missed the comment explaining the origin of the value